src/firewall/varktables/types.rs

```diff
@@ -353,6 +361,33 @@ pub fn get_network_chains<'a>(
         VarkChain::new(conn, FILTER.to_string(), NETAVARK_FORWARD.to_string(), None);
     netavark_forward_chain.create = true;
 
+    // Add NETAVARK_INPUT chain to INPUT chain
+    input_chain.build_rule(VarkRule {
+        rule: format!(
+            "-m comment --comment 'netavark firewall plugin rules' -j {}",
```

Comment:
This is the exact same string which is written here: https://github.com/containers/netavark/blob/main/src/firewall/varktables/types.rs#L343 , I think this should become a constant.

also should comment here just be called `netavark firewall rules` @Luap99 WDYT ?